duckduckgo/iOS
# Avoid AppTP DB initialization when disabled

Task/Issue URL: https://app.asana.com/0/1199333091098016/1205714053673490/f
Tech Design URL:
CC:

**Description**:

This PR disables AppTP database initialization when the feature is disabled.



**Steps to test this PR**:
1. Check that debug builds compile and run
2. Check that release builds compile and run
3. In a release build, check that the AppTP database isn't initialized - could do this by changing the Run configuration in the DuckDuckGo scheme to use `Release`, or run the app for Profiling since that also uses Release



**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
